packages/pwa/app/components/header/index.test.js

```diff
@@ -152,34 +152,6 @@ test('renders cart badge when basket is loaded', () => {
 })
 
 test('route to account page when an authenticated users click on account icon', async () => {
-    server.use(
-        rest.post('*/oauth2/login', (req, res, ctx) =>
-            res(ctx.delay(0), ctx.status(303), ctx.set('location', `/testcallback`))
-        ),
-        rest.get('*/testcallback', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.status(200))
-        }),
-
-        rest.post('*/oauth2/token', (req, res, ctx) =>
-            res(
-                ctx.delay(0),
-                ctx.json({
-                    customer_id: 'test',
-                    access_token: 'testtoken',
-                    refresh_token: 'testrefeshtoken',
-                    usid: 'testusid',
-                    enc_user_id: 'testEncUserId'
-                })
-            )
-        ),
-        rest.get('*/customers/:customerId', (req, res, ctx) =>
-            res(
-                ctx.json({
-                    ...mockedRegisteredCustomer
-                })
-            )
-        )
-    )
```

Comment:
We're doing something a bit different. Now with the refactor, the `/oauth2/login` returns a 200, whereas before, we're doing the dance of getting first a 303 and eventually the user details from `/oauth2/token`. 

It's a mock test, and I agree simpler is better.